proknow-sdk/Requestor.cs

```diff
@@ -298,17 +298,26 @@ internal async Task<string> GetDomainStatusAsync()
             IList<KeyValuePair<string, string>> headerKeyValuePairs = null, HttpContent content = null)
         {
             string hasMore = "true";
-            string next = null;
+            string nextPage = null;
             var patients = new List<string>();
+            var patientsRoute = route.StartsWith("/collections") ? false : true;
             while (hasMore == "true")
             {
-                if (next != null)
+                if (nextPage != null)
                 {
                     if (queryParameters == null)
                     {
                         queryParameters = new Dictionary<string, object>();
                     }
-                    queryParameters["next"] = next;
+                    if (patientsRoute)
+                    {
+                        queryParameters["page_newest"] = DateTime.UtcNow.ToString("o");
+                        queryParameters["page_number"] = nextPage;
+                    }
+                     else
```

Comment:
Indentation is 1 space too long

Reply:
Thanks for the catch - guess I got used to relying on auto-indentation in vscode